<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWIndow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resources/resources/img/battleshipsPozadina.png);</string>
      </property>
      <widget class="QLabel" name="labelNaslov">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lohit Gujarati</family>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
background-image: url(:/resources/resources/img/battleshipsNaslov.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/resources/resources/battleshipsNaslov.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSingleplayer">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>180</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Singleplayer </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonMultiplayer">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>300</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Multiplayer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRezultati">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>420</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Najbolji rezultati</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resources/resources/img/battleshipsPozadina.png);</string>
      </property>
      <widget class="QTableWidget" name="tableWidgetRezultati">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>224</width>
         <height>323</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Ime</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rezultat</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>28</red>
          <green>113</green>
          <blue>216</blue>
         </color>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="7" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="8" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item row="9" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButtonNazad_3">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>440</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>Nazad</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNajboljiRezultati">
       <property name="geometry">
        <rect>
         <x>291</x>
         <y>69</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:none;
color: rgb(255, 255, 255);
font: 700 20pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>Najbolji rezultati</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNaslov_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lohit Gujarati</family>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
background-image: url(:/resources/resources/img/battleshipsNaslov.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/resources/resources/battleshipsNaslov.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resources/resources/img/battleshipsPozadina.png);</string>
      </property>
      <widget class="QLabel" name="labelSingleplayer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
background-image: url(:/resources/resources/img/singleplayerNaslov.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/resources/resources/singleplayerNaslov.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>200</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none;
background-color: rgb(98, 160, 234);
font: 700 17pt &quot;Ubuntu&quot;;
color: rgb(17, 32, 89);</string>
       </property>
       <property name="text">
        <string>Easy</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none;
background-color: rgb(98, 160, 234);
font: 700 17pt &quot;Ubuntu&quot;;
color: rgb(17, 32, 89);</string>
       </property>
       <property name="text">
        <string>Hard</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTezina">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>140</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
color: rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Odaberite tezinu partije:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonZapocniPartiju">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>350</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Započni partiju</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonNazad">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>484</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nazad</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resources/resources/img/battleshipsPozadina.png);
</string>
      </property>
      <widget class="QLabel" name="labelMultiplayer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
background-image: url(:/resources/resources/img/multiplayerNaslov.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/resources/resources/multiplayerNaslov.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonZapocni">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>300</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Zapocni partiju</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditInputPort">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>200</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelPort">
       <property name="geometry">
        <rect>
         <x>343</x>
         <y>167</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 20pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string> Unesite IP:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPovezivanje">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>230</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Povezi se na server</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonNazad_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>490</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nazad</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
